scripts/romanisim-make-image

```diff
@@ -58,8 +60,27 @@ def go(args):
     else:
         persist = persistence.Persistence()
 
+    # Set up for extra counts as needed
+    extra_counts = None
+    if args.extra_counts is not None:
+        if len(args.extra_counts) == 1:
+            extra_counts_file = args.extra_counts[0]
+            extra_counts_hdu = 0
+        elif len(args.extra_counts) == 2:
+            extra_counts_file = args.extra_counts[0]
+            extra_counts_hdu = int(args.extra_counts[1])
+        else:
+            raise ValueError("If --extra-counts is used, must provide one or two arguments only.")
+        # Undocumented, but useful for testing 
+        if extra_counts_file=="_testing":
+            extra_counts = np.zeros((4088, 4088), dtype=np.float32)+float(extra_counts_hdu)
+        else:
+            with fits.open(extra_counts_file) as hdul:
+                extra_counts = hdul[extra_counts_hdu].data  
```

Comment:
```suggestion
        with fits.open(extra_counts_file) as hdul:
            extra_counts = hdul[extra_counts_hdu].data  
```

My inclination is to delete this.  If it got Poisson sampled it could be a hacky mechanism for changing the sky background & noise, but as is I have trouble seeing when people might want to use this.

Reply:
It's only here for me to test that the code runs, without having to create an extra file. Which is why I tried to bury it with an obscure call. 

But I'm happy to delete this, if you'd prefer.